hardhat.config.ts

```diff
@@ -1,39 +1,61 @@
-// hardhat.config.ts
+require("@nomiclabs/hardhat-waffle");
 
-import "dotenv/config"
-import "@nomiclabs/hardhat-etherscan"
-import "@nomiclabs/hardhat-solhint"
-import "@tenderly/hardhat-tenderly"
-import "@nomiclabs/hardhat-waffle"
-import "hardhat-abi-exporter"
-import "hardhat-deploy"
-import "hardhat-deploy-ethers"
-import "hardhat-gas-reporter"
-import "hardhat-spdx-license-identifier"
-import "hardhat-typechain"
-import "hardhat-watcher"
-import "solidity-coverage"
-import "./tasks"
+require('dotenv').config();
+import { resolve } from "path";
+import { config as dotenvConfig } from "dotenv";
+dotenvConfig({ path: resolve(__dirname, "./.env") });
 
-import { HardhatUserConfig } from "hardhat/types"
-import { removeConsoleLog } from "hardhat-preprocessor"
+const { PRIVATE_KEY, ETHERSCAN, POLYGONSCAN, FTMSCAN } = process.env;
 
-const accounts = {
-  mnemonic: process.env.MNEMONIC,
-  // accountsBalance: "990000000000000000000",
-}
-
-const config: HardhatUserConfig = {
+module.exports = {
   abiExporter: {
     path: "./abi",
     clear: false,
     flat: true,
-    // only: [],
-    // except: []
   },
   defaultNetwork: "hardhat",
+  networks: {
+    localhost: {
+      url: "http://127.0.0.1:8545",
+    },
+    hardhat: {
+      forking: {
+        enabled: process.env.FORKING === "true",
+        url: `https://eth-mainnet.alchemyapi.io/v2/${process.env.ALCHEMY_API_KEY}`,
+      },
+    },
+    mainnet: {
+      url: `https://mainnet.infura.io/v3/${process.env.INFURA_API_KEY}`,
+      accounts: [PRIVATE_KEY],
+      gasPrice: 120 * 1000000000,
+      chainId: 1,
+    },
+    ftm: {
+      // url: "https://rpc.ankr.com/fantom",
+      url: "https://rpcapi-tracing.fantom.network",
+      accounts: process.env.PRIVATE_KEY ? [process.env.PRIVATE_KEY] : undefined,
```

Comment:
Not imported ENV (COINMARKETCAP_API_KEY, INFURA_API_KEY, ALCHEMY_API_KEY)

Reply:
Removing COINMARKETCAP_API_KEY, ALCHEMY_API_KEY as unnecessary, adding INFURA_API_KEY to .env.example